fido/exceptions.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+
+
+class ConnectionError(Exception):
```

Comment:
Having "Connection" in the name is potentially confusing with "Connect" and "HTTP" used in the same context.  TBH, YAGNI - I see it isn't used in the CR for bravado_decorators.